src/Umbraco.Core/UmbracoEvents.cs

```diff
@@ -0,0 +1,10 @@
+using System;
+using Umbraco.Web.Routing;
+
+namespace Umbraco.Core
+{
+    public interface IUmbracoRouteEventSender
```

Comment:
Code comments? need to explain what this is

Reply:
I could remove this interface..